### ISSUE TITLE
Thunderbird: NVDA repeats the subject line three times when opening a message

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9247]
### Steps to reproduce:
1. Open Thunderbird
2. Open a message by pressing enter

### Actual behavior:
NVDA repeats the subject line (window title) three times before starting to report the message body.

### Expected behavior:
NVDA should report the subject line just once.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2018.4.1

#### Windows version:
Windows 8.1

#### Name and version of other software in use when reproducing the issue:
Thunderbird 60.

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
no

**Comment by Adriani90** [comment]
This is still reproducible in NVDA alpha-19420,a2c6f0f8
Thunderbird 68.3.0 and Windows 10 1909 update (build 18363.47).

**Comment by Adriani90** [comment]
Closing as duplicate of #1776.
